jerry-core/ecma/builtin-objects/ecma-builtin-helpers-error.c

```diff
@@ -45,7 +46,8 @@ ecma_builtin_helper_error_dispatch_call (ecma_standard_error_t error_type, /**<
   JERRY_ASSERT (arguments_list_len == 0 || arguments_list_p != NULL);
 
   if (arguments_list_len != 0
-      && !ecma_is_value_undefined (arguments_list_p[0]))
+      && !ecma_is_value_undefined (arguments_list_p[0])
+      && !ECMA_IS_VALUE_ERROR (arguments_list_p[0]))
```

Comment:
I don't think this is right. arguments_list_p[0] cannot hold an error, since an error is a "status" of the engine, not a value which can be passed around.

We should fix it before the ecma_builtin_helper_error_dispatch_call is called.